e2e/cypress/support/api/helpers.js

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+export function buildQueryString(queryParams = {}) {
+    let queryString = '';
+    Object.entries(queryParams).forEach(([k, v], index) => {
+        if (index > 0) {
+            queryString += '&';
+        }
+
+        queryString += `${k}=${encodeURIComponent(v)}`;
```

Comment:
Shouldn't key also be encoded?

Reply:
I don't think so and honestly don't know if we're doing it in other place. In mattermost-redux, we're only encoding the value as well.  Key is something we have control/validation in server and I don't think we allow a key that is unescaped or will require to be encoded. But happy to change if necessary.